readthedocs/projects/views/public.py

```diff
@@ -37,7 +37,7 @@
 mimetypes.add_type('application/epub+zip', '.epub')
 
 
-class ProjectIndex(ListView):
+class ProjectTagIndex(ListView):
```

Comment:
I'm just noting that this change will require a corresponding change to the private Read the Docs corporate repository.